lib/koans/15_protocols.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Protocols do
+  use Koans
+
+  defprotocol School do
+    def enrol(name)
```

Comment:
Hahahah. You'll notice this throughout the Koans. @gemcfadyen @RabeaGleissner @ukutaht @Maikon  and I are all based in London.